code/mob/living/critter/scuttlebot.dm

```diff
@@ -124,6 +129,28 @@
 
 	setup_hands()
 
+/mob/living/critter/robotic/scuttlebot/mail
+	name = "Carr1er P1G30N"
+	desc = "A pigeon that must've escaped from the ranch and been trained to deliver mail... wait why is 8G labeled on its leg?"
+	icon = 'icons/mob/critter/robotic/scuttlebot.dmi'
+	icon_state = "pigeon"
+	mail_glasses = TRUE
+	speech_verb_say = "tweets"
+	speech_verb_exclaim = "twoots"
+	speech_verb_ask = "tweets curiously"
+	var/obj/item/clothing/suit/pigeon/linked_pigeon = null
+
+	add_abilities = list(/datum/targetable/critter/control_owner/mail)
+
+	setup_hands()
+		..()
+		var/datum/handHolder/HH = hands[1]
+		HH.limb = new /datum/limb/small_critter/strong
```

Comment:
This should proobably not have stronger hands than a syndie scuttlebot. Usually I'd say hands with holding exceptions are dumb, but it makes sense here for this thing to be specialised for picking up mail. Make it a weaker hand with an exception for mail items, if you have trouble figuring out the code feel free to ping me or ask in #imcoder